snaxc/transforms/convert_tosa_to_kernel.py

```diff
@@ -58,11 +66,19 @@ def match_and_rewrite(self, rescale_op: tosa.RescaleOp, rewriter: PatternRewrite
         shift = rescale_op.shift.get_values()
         assert isa(shift, tuple[int, ...])
         if isinstance(clamp_op, tosa.ClampOp):
-            max_int = assert_int8(clamp_op.max_int.value.data)
-            min_int = assert_int8(clamp_op.min_int.value.data)
+            if rescale_op.output.type.element_type == builtin.i8:
+                max_int = assert_int8(clamp_op.max_int.value.data)
```

Comment:
xdsl already has this functionality
```suggestion
                i8.verify_value(max_int)
```